addons/website_sale/views/templates.xml

```diff
@@ -683,16 +695,29 @@
         </t>
     </template>
 
-    <template id="product_custom_text" inherit_id="website_sale.product" active="True" customize_show="True" name="Terms and Conditions">
-        <xpath expr="//div[@id='product_details']" position="inside">
-            <p class="text-muted">
-                <a href="/terms">Terms and Conditions</a><br/>
+    <template id="product_custom_text" inherit_id="website_sale.product" active="True" customize_show="True" name="Terms and Conditions" priority="21">
```

Comment:
Don't like those priority additions... one way would be to add an empty `<t>` element with an ID in the original template, at the right position, and replace it thanks to this option template. Not really prettier and it has its customization flaws as well... what do you think @JKE-be ?